SomosMasApp/SomosMasApp/App/Shared/Modules/LogIn/Service/LoginService.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  LoginService.swift
+//  SomosMasApp
+//
+//  Created by Mariano Battaglia on 26/03/2022.
+//
+
+import Foundation
+import Alamofire
+
+struct LoginService {
+
+    static let shared = LoginService()
+    let baseURL = "https://ongapi.alkemy.org/api/login"
+    let token = ""
+
+    func login(user: Credentials, completionHandler: @escaping ResultLoginHandler, errorHandler: @escaping ErrorHandler) {
+
+        let Encoder = JSONParameterEncoder.default
+        AF.request(baseURL, method: .post, parameters: user, encoder: Encoder).responseDecodable(of: LoginUserResponse.self, decoder: JSONDecoder()){ response in
+
+            switch response.result {
+            case .success(let data):
+                guard let token = data.data.token else {
+                    errorHandler(data.message)
+                    print(token)
```

Comment:
You should not have any prints on a PR if you want to log things this is not the way, is ok to do it as a helper to debug but not to commit them on a PR